README.md

```diff
@@ -156,38 +151,6 @@ There are three output states
 - [WARN] means this test needs further attention, for example it is a test that needs to be run manually 
 - [INFO] is informational output that needs no further action.
 
-## Test config YAML representation
```

Comment:
I think we should still have at least this heading and the initial sentence in the main readme, so people can easily see that tests are configured in YAML. And there should be a link to take them to the details. We probably need the example as well, because otherwise the next section on Omitting checks doesn't make sense.

Reply:
Done.